docs/data/gridftp.md

```diff
@@ -111,6 +111,26 @@ All other configuration settings should be suitable for most purposes. However,
 
 ### Optional configuration
 
+### Modifying the limits
```

Comment:
- Let's change this to "Setting transfer limits"
- This should be a level-4 header